group.py

```diff
@@ -71,25 +71,26 @@ def add_connection(self, friend, relationship):
 [print(member['age']) for member_name, member in group.items()]
 
 # 1. Max age of people in the group 
-ages = []
-[ages.append(member['age']) for member_name, member in group.items()]
-print("Maximum age of group: ", max(ages))
+#Not proper comprehensions --> Comprehensions make a list already 
+#ages = []
+#[ages.append(member['age']) for member_name, member in group.items()]
+
+#Making use of proper comprehensions
+ages = [person["age"] for person in group.values()]
+print("Maximum age of group: ", max(ages)) 
 
 # 2. Average (mean) number of relations among members of the group 
-no_relations = []
-[no_relations.append(len(member['relations'])) for member_name, member in group.items()]
+no_relations = [len(member['relations']) for member_name, member in group.items()]
```

Comment:
nice, you could also make a function that calculates the mean from a list and use that on the output of the list 

```python
def mean(data):
    """Compute the average of a non-empty list of numbers."""
    return sum(data) / len(data)

mean([len(person["relations"]) for person in group.values()])  # 1.5
```